features/bevragen/persoon/verblijfplaats/overzicht.feature

```diff
@@ -75,7 +75,7 @@ Rule: Een verblijfplaats is een onbekend verblijfplaats wanneer 'land adres buit
     | naam | waarde                 |
```

Comment:
Wat opmerkingen over de rest van dit bestand.

- Regel 7: Bevat de variabele '<gba veld>' die niet in de bijbehorende voorbeelden tabel voorkomt. Ik vermoed dat deze vervangen moet worden door 'straatnaam (11.10)'.
- Geldt voor de waarde in regel 27 niet dat er een type 'VerblijfplaatsOnbekend' gegenereerd moet worden? Nu wordt er volgens dit feature alleen maar een type 'VerblijfplaatsOnbekend' gegenereerd als 'land adres buitenland (13.10)' gelijk is aan 0000. Dat zou betekenen dat een binnelands adres nooit onbekend kan zijn.
- Regel 50: Bevat de variabele '<gba veld>' terwijl het scenario niet abstract is. Ik vermoed dat deze vervangen moet worden door 'land adres buitenland (13.10'.